src/webui/api/apierror.h

```diff
@@ -1,6 +1,6 @@
 /*
  * Bittorrent Client using Qt and libtorrent.
- * Copyright (C) 2014  Vladimir Golovnev <glassez@yandex.ru>
+ * Copyright (C) 2017  Vladimir Golovnev <glassez@yandex.ru>
```

Comment:
note: it's 2018 now.

Reply:
Really??? :)
Actually, some of files was created in 2017, the others in 2018.

Reply:
Changed all new to 2018.